students/GKim/lesson03/mailroom.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
+
+import sys
+import os
+
+main_donors = [("Luke Skywalker", [100, 200, 50]),
+          ("Han Solo", [100, 50, 600]),
+          ("Yoda", [1000, 50, 600, 200]),
+          ("Ben Kenobe", [10, 5, 6]),
+          ]
+
+def clear_screen():
+    """
+    clears the command screen
+    """
+    os.system("cls" if os.name == "nt" else "clear")
+
+
+def show_list():
+    """
+    shows the donors in the list without donation amounts
+    """
+    print()
```

Comment:
To avoid the excessive print() calls, you can add a "\n" at the beginning/end of the following/previous print

ex: print("\n" + "#" * 9, "The Current Donor List", "#" * 9)